test_scripts/upgrade.sh

```diff
@@ -10,6 +10,13 @@ then
   export END_BRANCH=$(git rev-parse --abbrev-ref HEAD)
 fi
 
+if [[ $START_BRANCH == 7* ]]
```

Comment:
You should check for >= 7.x so that it should work with 8.0 as well

Reply:
Thanks. Will make this change

Reply:
Should we use some other logic here to decide whether we have collections enabled or not, other than doing git branch?
Something like checking the existence of molecule directory at the top level, or maybe checking the existence of confluent. test directory inside roles.
Another option could be using .bumpversion.cfg file to get the cp-ansible version and decide on 
That way, even if someone is developing on a custom/local branch, then also our code would be able to confirm whether it's collection enabled or not.